### ISSUE TITLE
Installer crashes on Windows 10

### ISSUE THREAD

**Issue by sdaau** [Securepoint/openvpn-client#14]
I have downloaded the installer from https://sourceforge.net/projects/securepoint/files/openvpn-client-installer-2.0.24.exe/download

When I try to run it, I get this:

![sslvpn_inst_01](https://user-images.githubusercontent.com/26737749/53231877-325f3d80-368a-11e9-968b-b8e7d205189a.png)

I choose English, hit OK - then it says:

![sslvpn_inst_02](https://user-images.githubusercontent.com/26737749/53231903-3c813c00-368a-11e9-87b1-8852217178a7.png)

> Windows Installer    
> Invalid command-line argument. Consult the Windows Installer SDK for detailed command-line help. 

Not sure what to do about this; this is:

```
>systeminfo

OS Name:                   Microsoft Windows 10 Home
OS Version:                10.0.17763 N/A Build 17763
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
...
System Type:               x64-based PC
```

And in Windows About, I have:

> Edition: Windows 10 Home  
> Version: 1809  
> OS Build: 17763.316  

--------

EDIT: Note that the previous version of the installer, https://sourceforge.net/projects/securepoint/files/openvpn-client-installer-2.0.23.exe/download, works fine on this machine! However, once you try starting it from double-clicking the "Securepoint SSL VPN" icon, nothing starts up - not even when Run as Administrator. Only thing I could notice in Task Manager, is that when starting/double-clicking the icon, I get two "COM Surrogate" processes, that hang around for 3-5 seconds, and then they disappear. I also have McAfee LiveSafe on this machine, but I checked its logs, it looks like it doesn't block programs (or if it does, it does not keep an obvious log about it)

**Comment by ediazrod** [comment]
Happens to mee to..

**Comment by QtOlli** [comment]
I fixed this issue and upload a new installer(2.0.25) as soon as possible.

Thank you very much for reporting this issue!

Oliver

**Comment by QtOlli** [comment]
2.0.25 fix this issue. Sorry for your inconvenience!

kind regards
Oliver
